crates/solidity/inputs/language/src/definition.rs

```diff
@@ -6627,37 +6627,366 @@ codegen_language_macros::compile!(Language(
         ),
         BuiltInFunction(name = "assert", parameters = ["bool condition"]),
```

Comment:
side question: are any new global functions/variables bound exclusively from Solidity or Yul? not sure if we need to make that destinction.